### PR TITLE
Create blacklist.txt

### PR DESCRIPTION
**Pull request by Paddyk45**

### CODE REVIEW

**Comment by Juliandev02** [comment]
Thanks for your pull request. I'll review it asap.
I will definitely remove some of the words that are blacklisted, as I don't find them to be too much inappropriate/forbidden.

File: client/blacklist.txt [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+twink
+twinkie
+two girls one cup
+undressing
+upskirt
+urethra play
+urophilia
+vagina
+venus mound
+vibrator
```
Comment (matteodev8) [inline client/blacklist.txt:425]: :question:

Comment (Juliandev02) [reply client/blacklist.txt:425]: as i mentioned, i'll definitely remove some of the words that are blacklisted on this pr

**Review by Juliandev02 — APPROVED** [review]
👍

**Pull request merged**